.github/workflows/build-dotnet.yml

```diff
@@ -0,0 +1,76 @@
+name: Build .NET SDK
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  generate_schemas:
+    uses: ./.github/workflows/generate_schemas.yml
+
+  build_rust:
+    uses: ./.github/workflows/build-rust-cross-platform.yml
+
+  build_dotnet:
+    runs-on: ubuntu-22.04
+    needs: [generate_schemas, build_rust]
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Download c# schemas artifact
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: schemas.cs
+          path: languages/csharp/Bitwarden.Sdk
+
+      - name: Set up .NET Core
+        uses: actions/setup-dotnet@3447fd6a9f9e57506b15f895c5b76d3b197dc7c2 # v3.2.0
+        with:
+          global-json-file: languages/csharp/global.json
+
+      - name: Download macos-x64 files
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: libbitwarden_c_files-macos-x64
+          path: languages/csharp/Bitwarden.Sdk/macos-x64
+
+      - name: Download macos-arm64 files
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: libbitwarden_c_files-macos-arm64
+          path: languages/csharp/Bitwarden.Sdk/macos-arm64
+
+      - name: Download ubuntu-x64 files
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: libbitwarden_c_files-ubuntu-x64
+          path: languages/csharp/Bitwarden.Sdk/ubuntu-x64
+
+      - name: Download windows-x64 files
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        with:
+          name: libbitwarden_c_files-windows-x64
+          path: languages/csharp/Bitwarden.Sdk/windows-x64
```

Comment:
Not sure if it's just easier to download all files and add a `mv` command after instead of having multiple download steps.

Reply:
I would leave it for now like this, it is fast and clear if you are fine with that?